packages/@aws-cdk/aws-appmesh/lib/route.ts

```diff
@@ -61,17 +73,73 @@ export interface RouteProps extends RouteBaseProps {
   readonly virtualRouter: IVirtualRouter;
 }
 
+abstract class RouteBase extends cdk.Resource implements IRoute {
+  /**
+   * The name of the Route
+   *
+   * @attribute
+   */
```

Comment:
Same comment as in `GatewayRoute` (no reason to repeat JSDocs for overridden members).

Notice for example that in `MeshBase`, the `grant*()` methods don't have JSDocs repeated.